content/ConnectingToDatabase/ConnectingToDatabase.q.md

```diff
@@ -0,0 +1,673 @@
+        if($conn){
+            echo "Successfully connected!"; 
+        }
+    }    
+    catch(Exception $e){
+        echo "Unable to establish a connection.";
+    }
+?>
+```
+/// type=SS, answer=[3]
+
+Execute the program. What is its output?
+
+- It produces an error.
+
+- No output is displayed.
+
+- It prints `Successfully connected!`.
+
+- It prints `"Successfully connected"`.
+
+- It prints `Unable to establish a connection.`.
+
+
+/// type=SS, answer=[3]
+
+On line 2, what is `$host`?
+
+- It is a string.
+
+- It is a function.
+
+- It is a variable.
+
+- It is a statement.
+
+- It is an operator.
+
+
+/// type=SS, answer=[2]
+
+In the statement `$host='localhost';` on line 2, what does the assignment operator `=` do?
+
+- It displays the value assigned to the `$host` variable.
+
+- It assigns the `localhost` value to the `$host` variable.
+
+- It assigns the `$host` variable to the `localhost` value.
+
+- It replaces the `localhost` value with the `$host` variable.
+
+- It removes the `localhost` value from the `$host` variable.
+
+
+/// type=SS, answer=[5]
+
+In the statement `$dsn = "pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password";`, what is `pgsql:`?
+
+- It is the object.
+
+- It is the `PDO` class.
+
+- It is the username.
+
+- It is the hostname.
+
+- It is the DSN prefix.
+
+
+/// type=SS, answer=[1]
+
+Which variable contains the value of the database name?
+
+- `$db`
+
+- `$dsn`
+
+- `$port`
+
+- `$host`
+
+- `$username`
+
+
+/// type=SS, answer=[4]
+
+Which variable contains the value of the database server's hostname?
+
+- `$db`
+
+- `$dsn`
+
+- `$port`
+
+- `$host`
+
+- `$username`
+
+
+/// type=SS, answer=[5]
+
+What is the name of the database?
+
+- `5432`
+
+- `Admin01`
+
+- `postgres`
+
+- `localhost`
+
+- `MyDatabase`
+
+
+/// type=SS, answer=[5]
+
+On line 4, what is `5432`?
+
+- It is the password.
+
+- It is the DSN prefix.
+
+- It is the database name.
+
+- It is the server’s hostname of the PostgreSQL database.
+
+- It is the default port on which PostgreSQL database server is running on.
+
+
+/// type=SS, answer=[5]
+
+In the statement `$dsn = "pgsql:host=$host;port=$port;dbname=$db;user=$username;password=$password";`, what does the `$dsn` do?
+
+- It creates the object.
+
+- It holds the value of the user's password.
+
+- It contains the value of the database name.
+
+- It stores the value of the DSN prefix `pgsql:`.
+
+- It holds the value of the `PDO_PGQSL DSN` statement.
+
+
+/// type=SS, answer=[4]
+
+Which of the following is an object?
+
+- `new`
+
+- `$dsn`
+
+- `$host`
+
+- `$conn`
+
+- `$dbname`
+
+
+/// type=MS, answer=[4,5]
+
+Which statements correctly describe the code on line 11?
+
+- It assigns `PDO($dsn)` to `$conn`.
+
+- It sets the value of `$dsn` to `PDO()`.
+
+- It evaluates the `$conn` object of the `PDO` class.
+
+- It creates the `$conn` object instance of the `PDO` class.
+
+- It passes the variable `$dsn` as an argument to the `$conn` object.
```

Comment:
Please rephrase this one. I think `$dsn` is an argument of the `PDO` class. Please check. Thanks.